--- src/mainwindow.cpp
+++ src/mainwindow.cpp
@@ -342,8 +342,7 @@
           &MainWindow::showNewSessionDialog);
   actionCollection()->addAction(QStringLiteral("new_session"),
                                 newSessionAction);
-  KGlobalAccel::setGlobalShortcut(newSessionAction,
-                                  QKeySequence(Qt::CTRL + Qt::ALT + Qt::Key_N));
+  actionCollection()->setDefaultShortcut(newSessionAction, QKeySequence(Qt::CTRL + Qt::Key_N));

   m_showFullSessionListAction =
       new QAction(i18n("Show Full Session List"), this);
@@ -403,8 +402,7 @@
           &MainWindow::toggleWindowVisibility);
   actionCollection()->addAction(QStringLiteral("toggle_window"),
                                 toggleWindowAction);
-  KGlobalAccel::setGlobalShortcut(toggleWindowAction, QKeySequence());
+  actionCollection()->setDefaultShortcut(toggleWindowAction, QKeySequence(Qt::CTRL + Qt::Key_M));

   m_viewSessionsAction =
       new QAction(QIcon::fromTheme(QStringLiteral("view-list-details")),
