--- src/filtereditor.cpp
+++ src/filtereditor.cpp
@@ -461,11 +461,9 @@
 bool FilterEditor::eventFilter(QObject *obj, QEvent *event) {
   if (obj == m_lineEdit) {
     if (event->type() == QEvent::FocusIn) {
-      if (m_lineEdit->text().startsWith(QLatin1String("="))) {
-        m_userDismissed = false;
-        updatePopupPosition();
-        m_popupFrame->show();
-        m_toggleButton->setChecked(true);
-      }
+      m_userDismissed = false;
+      updatePopupPosition();
+      m_popupFrame->show();
+      m_toggleButton->setChecked(true);
     } else if (event->type() == QEvent::FocusOut) {
       QWidget *reasonWidget = QApplication::focusWidget();
