--- src/refreshprogresswindow.cpp
+++ src/refreshprogresswindow.cpp
@@ -110,13 +110,9 @@
   if (m_activeTasks.contains(id)) {
     m_activeTasks.remove(id);

     // Remove the id from the multimap
-#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
-    QMutableMultiMapIterator<QString, QString> i(m_activeTasksPrUrls);
-#else
-    QMutableMapIterator<QString, QString> i(m_activeTasksPrUrls);
-#endif
-    while (i.hasNext()) {
-      i.next();
-      if (i.value() == id) {
-        i.remove();
-      }
-    }
+    for (auto it = m_activeTasksPrUrls.begin(); it != m_activeTasksPrUrls.end();) {
+      if (it.value() == id) {
+        it = m_activeTasksPrUrls.erase(it);
+      } else {
+        ++it;
+      }
+    }

     m_processedCount++;
     m_progressBar->setValue(m_processedCount);
