--- src/activitybrowser.cpp
+++ src/activitybrowser.cpp
@@ -19,6 +19,10 @@
 #include <QScreen>
 #include <QVBoxLayout>

+static QString closeTurnBlock() {
+  return QStringLiteral("<div style='clear: both;'></div></div>");
+}
+
 ActivityBrowser::ActivityBrowser(QWidget *parent) : QTextBrowser(parent) {
   setOpenLinks(false);
   setContextMenuPolicy(Qt::CustomContextMenu);
@@ -246,8 +250,7 @@
         html += generateHtmlForActivity(activity, expanded);
       }

-      html += QStringLiteral("<div style='clear: both;'></div>");
-      html += QStringLiteral("</div>");
+      html += closeTurnBlock();
     }
   }
