--- src/mainwindow.cpp
+++ src/mainwindow.cpp
@@ -2224,9 +2224,12 @@
 }

 void MainWindow::updateCountdownStatus() {
+  // Inform the queue model to update display of active wait items, even if paused
+  m_queueModel->refreshWaitItems();
+
   if (m_queuePaused || m_queueModel->isEmpty()) {
     return;
   }

   QDateTime now = QDateTime::currentDateTimeUtc();
   qint64 secondsLeft = 0;
@@ -2261,9 +2264,6 @@
     }
     updateStatus(i18n("Next attempt in %1...", timeStr));
   }
-
-  // Inform the queue model to update display of active wait items
-  m_queueModel->refreshWaitItems();
 }

 void MainWindow::toggleQueueState() {
